snowflake/Integration.txt

```diff
@@ -0,0 +1,46 @@
+-- creating database and warehouse --
+
+use role sysadmin;
+
+create database if not exists playdb;
+
+create warehouse if not exists playwh with warehouse_size='xsmall' auto_suspend=60 initially_suspended=true;
+
+use schema playdb.public;
+
+-- cloud storage integration --
+
+use role accountadmin;
+
+create storage integration azure_snowflake_integration_171_2
+      type = external_stage
+      storage_provider = azure
+      enabled = true
+      azure_tenant_id = 'tenant_id'
```

Comment:
Let's unify the way you mask secure strings. For `azure_tenant_id` you used `tenant_id`, for `storage_allowed_locations` you used `*`. I ok with with both methods, but let's be consequent.